boilerplate/lyft/end2end/end2end.sh

```diff
@@ -13,21 +13,17 @@ OUT="${DIR}/tmp"
 rm -rf ${OUT}
 git clone https://github.com/lyft/flyte.git "${OUT}"
 
-echo "Loading github docker images into 'kind' cluster to workaround this issue: https://github.com/containerd/containerd/issues/3291#issuecomment-631746985"
```

Comment:
is this no longer applicable? i see we're still using kind, right?

Reply:
We are using kind but that specific issue was related to pulling anonymous images from github docker registry... which is the part I got rid of

Reply:
thanks for explaining!